cnchi/installation/install.py

```diff
@@ -774,10 +774,10 @@ def update_pacman_conf():
                     else:
```

Comment:
@lots0logs It's still after testing, it shouldn't be since the main priority is Antergos.  If testing was used staging would be used at the same time.